File: PHP/BrinquedosDoPapaiNoelTest.php
<?php
/**
 * @version PHP
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ ./vendor/bin/phpunit NomeDaClasseTest.php
 */
namespace PHP;

use PHPUnit\Framework\TestCase;
require ("BrinquedosDoPapaiNoel.php");

class BrinquedosDoPapaiNoelTest extends TestCase {

    public function testBrinquedosDoPapaiNoel() {
        $obj = new BrinquedosDoPapaiNoel();
        //Test 1
        $want = [2, 3];
        $got = $obj->howManyToys( ["Milena F", "Joao M", "Rafaela F", "Renata F", "Felipe M"] );
        $this->assertEquals($want, $got);
    }

}

File: GO/BrinquedosDoPapaiNoel_test.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ go test
 */
package GO

import (
	"reflect"
	"testing"
)

func TestBrinquedosDoPapaiNoel(t *testing.T) {
	tables := []struct {
		inputs []string
		want   []int
	}{
		{[]string{"Milena F", "Joao M", "Rafaela F", "Renata F", "Felipe M"}, []int{2, 3}},
	}

	for _, table := range tables {
		got := howManyToys(table.inputs)

		if !reflect.DeepEqual(got, table.want) {
			t.Errorf("Waiting for this %v but the return was this: %v", table.want, got)
		}
	}
}
